addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4222,7 +4255,12 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
             } else {
                 shapeToSelect = possibleShapes[1];
             }
-            return this._handlePreviewState(false, () => ({shape: shapeToSelect}));
+            _.defer(() => {
```

Comment:
Use setTimeout, avoid using underscorejs when you can

Reply:
Ultimately using a snippet_edition_request instead